tests/js/spec/views/settings/components/dataScrubbing/content.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import Content from 'app/views/settings/components/dataScrubbing/content';
+import {
+  Rule,
+  RuleType,
+  MethodType,
+} from 'app/views/settings/components/dataScrubbing/types';
+
+const rules: Array<Rule> = [
+  {
+    id: 0,
+    method: MethodType.MASK,
+    type: RuleType.CREDITCARD,
+    source: '$message',
+  },
+  {
+    id: 1,
+    method: MethodType.REPLACE,
+    placeholder: 'Scrubbed',
+    type: RuleType.PASSWORD,
+    source: 'password',
+  },
+];
+
+const handleEditRule = jest.fn();
+const handleDelete = jest.fn();
+
+describe('Content', () => {
+  it('default render - empty', () => {
+    const wrapper = mountWithTheme(
+      <Content
+        rules={[]}
+        onUpdateRule={handleEditRule}
+        onDeleteRule={handleDelete}
+        errors={{}}
+      />
+    );
+    expect(wrapper.text()).toEqual('You have no data scrubbing rules');
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Do we need the snapshot? I think the above assertion is enough

Reply:
sure.